core/controlplane/config/templates/cluster.yaml

```diff
@@ -1071,6 +1071,43 @@ addons:
 #   # When enabled, `kubectl drain` is run on the node going to shut-down
 #   nodeDrainer:
 #     enabled: true
+#   # Enable dex  integration - https://github.com/coreos/dex
+#   # Configure OpenID Connect token authenticator plugin in Kubernetes API server. Notice: always use "https" for the "url",
+#   # otherwise the Kubernetes API server will not start correctly. The "caFile" can be skipped if you use a certificate signed
+#   # by a trusted CA.
+#   dex:
+#     enabled: true
+#     url: "https://dex.example.com"
```

Comment:
After reading @c-knowles's comment from #509, I think using helm in the future will be a better option to keep `kube-aws`code clean. So, i think is better to provide some examples and a script to easily expose dex service using ELB or Ingress after the cluster is up.  Already made the commit. As for the Route53 record, I'm always use a kops's dns-controller to configure all the records automatically for Ingress and ELBs, but there are many solutions for this.